radio/build.rs

```diff
@@ -0,0 +1,39 @@
+extern crate bindgen;
+
+use std::fs;
+use std::path::PathBuf;
+
+fn main() {
+    // Tell cargo to look for shared libraries in the specified directory
+    //println!("cargo:rustc-link-search=/path/to/lib");
+
+    // Tell cargo to tell rustc to link the system bzip2
+    // shared library.
+    //println!("cargo:rustc-link-lib=bz2");
+
+    // Tell cargo to invalidate the built crate whenever the wrapper changes
+    println!("cargo:rerun-if-changed=include/wrapper.h");
```

Comment:
Yeah I actually thought about this this weekend a bit. I think if we end up having multiple bindgen artifacts we will just run bindgen multiple times to avoid this caching problem. The official guide suggested doing it this way... which seems bad

Reply:
Having a `wrapper.h` is probably fine and might be the right way if that's what the guide suggests but we'd want to at least change `"cargo:rerun-if-changed=include/wrapper.h"` to `"cargo:rerun-if-changed=include/"` so it will rerun if either of the header files change.

Although it seems like having a separate bindgen for each header file so each header is it's own module for the library might be a nicer structure.

Reply:
I did check the docs and the flag behaves the sane way if you point it to a directory.